GovUk.Frontend.Umbraco.ExampleApp/uSync/v9/Domains/1301_cy.config

```diff
@@ -0,0 +1,2 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Empty Key="433040cc-8ea1-41c4-8e6a-05083efd6c07" Alias="*1301" Change="Rename" />
```

Comment:
This represents a version you worked on and then deleted. When you're ready to PR again, do Settings > uSync > Everything > Export > Clean export to get rid of files like this one.